src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -270,12 +266,8 @@ export class DictionaryModal extends React.Component {
                       onChange={this.onChange}
                       value={data.public_access}
                     >
-                      <option value="" />
-                      {(isEditingDictionary && dictionary.public_access === 'None') &&
```

Comment:
Can you explain why this logic was taken away instead of simply changing the option label from None to Private?

Reply:
Kindly let me look into this issue before I can give you my final conclusion.

Reply:
Please correct me in case I am wrong, but after my review, I do not think that this block of code is necessary due to the fact that the default value on the select element is `public`. One can still choose one of the options when editing a dictionary.